### DISCUSSION TITLE
Development of an Internet of Things (IoT) Enabled Novel Wireless Multi Sensor Network for Infield Crop Monitoring

### DISCUSSION THREAD

**Discussion by nipuna-chamara** [STELLA-Landsat/STELLA#53] [Literature]
Here is a user case using the spectral triad for continuous crop monitoring. This in-field crop monitoring network has 10 down-looking special triads and one up-looking spectral triad. This system helps to continuously monitor and track the change in vegetation.
[https://digitalcommons.unl.edu/biosysengdiss/111/]

**Comment by Landsat-Mike** [comment]
Excellent! Thank you for sharing!!!
